targets/targets.json

```diff
@@ -571,6 +571,20 @@
         "release_versions": ["2", "5"],
         "device_name": "MKW41Z512xxx4"
     },
+    "RO359B": {
```

Comment:
why this target does not have MCU_K24F parent ? If there is a new target, it would  copy-paste lot of common code?